engine/api/workflow_run_test.go

```diff
@@ -97,14 +98,14 @@ func Test_getWorkflowNodeRunHistoryHandler(t *testing.T) {
 	w1, err := workflow.Load(db, api.Cache, proj, "test_1", u, workflow.LoadOptions{})
 	test.NoError(t, err)
 
-	wr, _, errMR := workflow.ManualRun(nil, db, db, api.Cache, proj, w1, &sdk.WorkflowNodeRunManual{
+	wr, _, errMR := workflow.ManualRun(nil, db, api.Cache, proj, w1, &sdk.WorkflowNodeRunManual{
```

Comment:
do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use